mobile/src/src/components/Community/Feed/EventView.jsx

```diff
@@ -64,12 +65,8 @@ const EventView = ({
             : "N/A"}
         </Paragraph>
 
-        {photo && (
-          <Image
-            source={{ uri: photo }}
-            style={{ width: 0.88 * width, height: 300 }}
-          />
-        )}
+        {photos && <ImageCollection photos={photos}> </ImageCollection>}
```

Comment:
If there are no children for the ImageCollection, then no need to have a closing tag.